playground/mock/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CYANS UI PLAYGROUND</title>
+  <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.rawgit.com/necolas/normalize.css/73b6b0c7/normalize.css">
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <main class="Playground">
+    <aside class="PlaygroundSidebar">
+      <h1 class="PlaygroundTitle">CYANS UI PLAYGROUND</h1>
+
+      <div class="Menu">
+        <div class="Folding">
+          <span class="Folding__Heading">Welcome</span>
+        </div>
+
+        <div class="Folding">
+          <span class="Folding__Heading">Button</span>
+          <span class="Folding__Subitem" aria-selected>with text</span>
```

Comment:
Beh nel caso del selected si, ma riesci ad avere stati per gni cosa? Perchè per anche solo una cosa che non c'è, se crei un modificatore sei anti-pattern.

Perche non usare i data attribute invece, cosi possiamo creare tutti i selettori che vogliamo?
Ad esempio se hai bisogno di uno stato `is-animating` che cambia rispetto allo stato finale del componente?

È solo una domanda eh... Se dici che non c'è il problema per me è ok. Solo mi piace trovare eventuali futuri problema! :D